media-libs/zxing-cpp/zxing-cpp-2.2.1-r1.ebuild

```diff
@@ -43,6 +44,7 @@ src_configure() {
 		-DBUILD_EXAMPLES=OFF # nothing is installed
 		-DBUILD_BLACKBOX_TESTS=$(usex test)
 		-DBUILD_UNIT_TESTS=$(usex test)
+		-DBUILD_DEPENDENCIES=LOCAL # force find_package as REQUIRED
```

Comment:
I'd considered doing this and didn't for some reason, I don't recall why.

Reply:
Ah, I think it's because there's some related new options in master and they're not in this release, and then I assumed this option wasn't in the older release at all.